### ISSUE TITLE
Should Screen.activeProperty initialize to false?

### ISSUE THREAD

**Issue by zepumph** [phetsims/joist#483]
Right now, in the constructor for `Screen`, the activeProperty is initialized to false. Then these screens are passed into Sim.js where their activeProperties are set according. This makes for a minor, but inconsistent initial state for Screen.activeProperty. For example, a single screen sim will still have the initial value of `screen.activeProperty` false.

**First Question**. . . Is this important? Does it matter? 

It shows up like this in phet-io studio, which isn't ideal. In https://github.com/phetsims/faradays-law/issues/107 we talked about making it default to true, but then still there is inconsistencies for all screens that are then toggled to inactive (n-1 screens for a sim).

**Next Question** . . . if this is important enough to fix, how to fix it?

There are three possible solutions @samreid came up with. 
1. Initializing to false.
2. Initializing to true.
3. Remove the `activeProperty` from the constructor, and move it to an `initialize` function. Then we can create a `setActive` function on `Screen` that will check if initialized, and if not, create the property, setting it to the correct value on startup, and if so, then just act like `Property.setValue()`.

More discussion is needed. @samreid can you illustrate your issues with seeing an incorrect initial value in phet-io studio? Below is what it looks like:

![image](https://user-images.githubusercontent.com/6856943/38950618-89ab2036-42f2-11e8-806c-32005b1f2555.png)

Note that in https://github.com/phetsims/faradays-law/issues/107 we have been playing around with this property, and as of this writing it has been renamed (soon to be reverted) to screenActiveProperty, and it is currently defaulting to true (soon to be reverted). The above capture ignores these inconsistencies.

**Comment by samreid** [comment]
One possibility--can Sim.js create the `screen.activeProperty` instead  of having it created in the Screen constructor?

**Comment by samreid** [comment]
Since the initial value of `Screen.activeProperty` was changed from false to true in cf702a6614434c53cbf4ef0abb280462bcd6fb4e, there is no longer any problem for single-screen sims.  

For Beer's Law Lab, we saw this in the data stream:

```
66 beersLawLab.concentrationScreen.activeProperty changed {"oldValue":true,"newValue":false}
67 beersLawLab.beersLawScreen.activeProperty changed {"oldValue":true,"newValue":false}
```

And we don't think it will be confusing at all to have incorrectly true initial value for these properties in studio.  Moving these properties to Sim.js would be much too complicated, we decided to close this issue.
